packages/keystone/redis.spec.js

```diff
@@ -0,0 +1,79 @@
+const Redis = require('ioredis')
+
+const conf = require('@open-condo/config')
+
+
+describe('Redis adapter', () => {
+    let redisClient
+    let nonPrefixedClient
+
+    beforeAll(() => {
+        jest.resetModules()
+        process.env.REDIS_URL = conf['REDIS_URL'] || 'redis://127.0.0.1:6379'
+        process.env.REDIS_FALLBACK_ENABLED = conf['REDIS_FALLBACK_ENABLED'] || 'true'
```

Comment:
Should this be resetted in afterAll or not?

Reply:
it's not necessary, but it won't hurt